### PR TITLE
Adds vcmp disk api

### LINKED ISSUE #1283: add vcmp disk api to sdk
api does not exist. add it

### PR DESCRIPTION
**Pull request by caphrim007**
Issues:
Fixes #1283

Problem:
The api did not exist

Analysis:
this adds it

Tests:
functional
unit

### CODE REVIEW

File: f5/bigip/tm/vcmp/test/functional/test_virtual_disk.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Copyright 2016 F5 Networks Inc.
```
Comment (jasonrahm) [inline f5/bigip/tm/vcmp/test/functional/test_virtual_disk.py:1]: copyright 2017

File: f5/bigip/tm/vcmp/virtual_disk.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# coding=utf-8
+#
+# Copyright 2016 F5 Networks Inc.
```
Comment (jasonrahm) [inline f5/bigip/tm/vcmp/virtual_disk.py:3]: copyright 2017

**Review by jasonrahm — APPROVED** [review]
lgtm

**Comment by pjbreaux** [comment]
@caphrim007 @jasonrahm: sorry guys, I was on PTO yesterday.

**Comment by caphrim007** [comment]
@pjbreaux it was my fault, I clicked your name yesterday, then realized my mistake and unclicked it.

**Pull request merged**